src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -9,6 +9,45 @@ public class RegistrationServiceImpl implements RegistrationService {
 
     @Override
     public User register(User user) {
+        if(userValidation(user)){
+            return storageDao.add(user);
+        }
         return null;
```

Comment:
The `register` method returns `null` if validation fails. Consider using exceptions or a result object to provide more information about why registration failed, which can improve error handling and user feedback.